client_admin/test/unit/test_auth.py

```diff
@@ -55,29 +54,92 @@ def test_invalid_input(self):
         self.assertRaises(ValueError, self.user_section.search, x=2)
 
 
-class TestAuthUser(base_builtins.PulpClientTests):
+class TestAuthUser(unittest.TestCase):
 
-    USER_LOGIN = 'test-login'
-    PASSWORD = 'test-password'
-    NAME = 'test-name'
+    def test__prompt_password_given(self):
+        """
+        Test that prompt password will ask for a password until the password
+        is not blank
```

Comment:
This docstring doesn't seem accurate for what this test is doing. It looks like the tests just makes sure the password works in the normal case.